### ISSUE TITLE
furnace don't heat

### ISSUE THREAD

**Issue by sbuisson** [Glitchfiend/ToughAsNails#101]
you can froze to death near an active furnace

**Comment by hariedo** [comment]
Also open flames (burning netherrack) seem to have no warming effect.  Any of the flame bricks should have some heat buildup.  If it's challenging to calculate it yourself, using the artificial light level might be suitable.  Then glowmoss and redstone would even have a weak heating effect.

**Comment by Scootty** [comment]
I can share some notes on this with you guys. From what I've found, it's highly recommended that you sleep through the night or always wear Wool Armor as it has ~about a 30% warmth increase and most of the time will prevent you from freezing to death. I wouldn't recommend staying out until midnight though (When the moon is in the middle dial of the clock) as that is the coldest point in time and can kill you via Hypothermia regardless. Also, do not jump in any water, not sure how the wetness factor works or how long it lasts.

Other things that warm you up that I've found so far are the following: Campfire (~50%; Effect stacks), Fire such as those made from flint and steel (Minimal effects but 7 maxed out the temp at midnight), Lava (being near 5 source blocks maxed out the temp at midnight), best thing is obviously the Heating Coil. Campfires are nice but they eat up your sticks just to light, I'd seriously recommend the dev makes it max out the amount of sticks between 1-5 or so as I've gone through half stacks just trying to light one of those.
